### PR TITLE
Network scalability

### PR DESCRIPTION
**Pull request by paigerube14**
Using 1 namespace/deployment and scaling up and down

passing job: [https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-mu[…]line/job/regression-test/506/console](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/regression-test/506/console)

### CODE REVIEW

**Comment by svetsa-rh** [comment]
Looks good to me. Will let @qiliRedHat or @SachinNinganure review this one.

File: perfscale_regression_ci/scripts/network/network_policy_scalability.sh [hunk 1]
```diff
@@ -17,66 +17,70 @@ source ../common.sh
...
 check_http_code $return_traffic_code $pod_ip $apiserver_pod
 return_traffic_end=`date +%s`
 return_traffic_final_time=$((return_traffic_end - return_traffic_start))
 echo "Time taken for network policy to become active: $return_traffic_final_time s"
 
-echo "=========================================================================="
-
 
 final_time_np=${final_time_list[0]}
 final_time_no_np=${final_time_list[1]}
```
Comment (qiliRedHat) [inline perfscale_regression_ci/scripts/network/network_policy_scalability.sh:86]: @paigerube14 Why do you want to remove the "no-np-issue-test"? If it is removed, I think this line should be removed too.

File: perfscale_regression_ci/scripts/network/network_policy_scalability.sh [hunk 2]
```diff
@@ -100,7 +104,6 @@ else
 	echo "Time when network traffic returns: $return_traffic_final_time s"
 fi
 
-
 policy_no_policy_difference=$(calculate_difference ${final_time_np} ${final_time_no_np})
```
Comment (qiliRedHat) [inline perfscale_regression_ci/scripts/network/network_policy_scalability.sh:107]: If "no-np-issue-test" is removed, ${final_time_no_np} should be removed too.

File: perfscale_regression_ci/scripts/network/network_policy_scalability.sh [hunk 3]
```diff
@@ -17,66 +17,70 @@ source ../common.sh
...
 echo "======Use kube-burner to load the cluster with test objects======"
 
+echo "Starting scale-up test for $workload_template..."
+export NAME=${kube_burner_job_list}
+export NAMESPACE=${kube_burner_job_list}
+deployment_name=${kube_burner_job_list}
+export WORKLOAD_TEMPLATE=$workload_template
+run_workload
+
 for ((i = 0; i < 2; i++));
```
Comment (qiliRedHat) [inline perfscale_regression_ci/scripts/network/network_policy_scalability.sh:43]: It seems the for loop is not needed here anymore if kube_burner_job_list has only one item.

**Pull request closed without merging**